flaskapp/dataset.py

```diff
@@ -1,22 +1,22 @@
 import json
 import os
 
-class Dataset:
 
-    def __init__(self, name='dataset'):
+class Dataset:
+    def __init__(self, name="dataset"):
```

Comment:
This should use ANNOTATIONS_PATH from constants. It is already added in the repo with some default queries.